client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala

```diff
@@ -151,6 +151,10 @@ object CapiModelEnrichment {
         "0d18e8413ab7cdf377e1202d24452e63"
       )
 
+      val specialReportAltTags: Set[String] = Set(
```

Comment:
Knowing that this needs to be salted first, maybe we just use `hashedSpecialReportAltTags`

Reply:
Done!